addons/mail/static/src/model/model_field.js

```diff
@@ -272,13 +272,12 @@ class ModelField {
             return this.read(record);
         }
         if (this.fieldType === 'relation') {
-            const OtherModel = this.env.models[this.to];
             if (['one2one', 'many2one'].includes(this.relationType)) {
-                return OtherModel.get(this.read(record));
+                return this.read(record);
             }
             const res = [];
-            for (const localId of this.read(record)) {
-                res.push(OtherModel.get(localId));
+            for (const otherRecord of this.read(record)) {
+                res.push(otherRecord);
             }
             return res;
```

Comment:
basically `return [...this.read(record)]`

Reply:
Method is meant to be removed in following commit, I when just for the minimal/mechanical diff here. Could indeed be simplified.